src/scripting/game_lua_kernel.cpp

```diff
@@ -4197,10 +4169,18 @@ game_lua_kernel::game_lua_kernel(game_state & gs, play_controller & pc, reports
 	lua_setfield(L, -2, "current");
 	lua_pop(L, 1);
 
-	// Add tovconfig to the WML module
+	// Add functions to the WML module
 	lua_getglobal(L, "wml");
-	lua_pushcfunction(L, &lua_common::intf_tovconfig);
-	lua_setfield(L, -2, "tovconfig");
+	static luaL_Reg const wml_callbacks[] {
+		{"tovconfig", &lua_common::intf_tovconfig},
+		{"eval_conditional", &intf_eval_conditional},
+		// These aren't actually part of the API - they're used internally by the variable metatable.
+		{ "get_variable", &dispatch<&game_lua_kernel::intf_get_variable>},
+		{ "set_variable", &dispatch<&game_lua_kernel::intf_set_variable>},
+		{ "get_all_vars", &dispatch<&game_lua_kernel::intf_get_all_vars>},
+		{ nullptr, nullptr }
```

Comment:
Having all of the commits in mainline be self-contained is helpful when in future a developer is using `git bisect`.

There's currently a 10-commit window where this table lacks its sentinel - when rebasing, please squash fix commits like this into the commit that needs them.